app.js

```diff
@@ -17,15 +18,13 @@ const adminRouter = require('./admin/admin.router');
 // Utils
 const utils = require('./utils/utils');
 
-const AUTH_TOKEN = process.env.SPLIT_EVALUATOR_AUTH_TOKEN;
-
 app.use(morgan('tiny'));
 
 // OPENAPI 3.0 Definition
 // Grabs yaml
 const openApiDefinition = YAML.load(fs.readFileSync('./openapi/openapi.yaml').toString());
 // Informs warn and remove security tag
-if (!AUTH_TOKEN) {
+if (!environmentManager.getAuthTokens()) {
```

Comment:
Emma, be careful here. I think this condition is always `false`, because `getAuthTokens` always returns an array that might be empty or not.

Reply:
`if (!environmentManager.getAuthTokens().length)`

Reply:
nice catch!